CHANGELOG.md

```diff
@@ -6,10 +6,8 @@ All notable changes to this project will be documented in this file.
 
 ### Added 
 - Add the initial Xcode project.
-
-## [0.1.0] - 2023-03-05
-
-### Added 
 - Add the Swiftlint script to the build phase of the Woof target.
 - Add the Swiftformat script to the build phase of the Woof target.
 - Update the deployment target to 15.0. 
+
+## [0.1.0] - 2023-03-05
```

Comment:
Maybe we can remove the exact date when the version will be officially changed.

Reply:
The exact date removed.